src/main/docs/guide/views/fieldset/fieldsetExample.adoc

```diff
@@ -5,11 +5,11 @@ Imagine, you want to create an application which displays a form such as:
 <form action="/books/save" method="post">
     <div class="mb-3">
         <label for="title" class="form-label">Title</label>
-        <input type="text" name="title" value="" id="title" class="form-control" required="required"/>
+        <input type="text" name="title" value="" id="title" minlength="2" maxlength="255" class="form-control" required="required"/>
     </div>
     <div class="mb-3">
         <label for="pages" class="form-label">Pages</label>
-        <input type="number" name="pages" value="" id="pages" class="form-control" required="required"/>
+        <input type="number" name="pages" value="" id="pages" min="1" max="21450" class="form-control" required="required"/>
```

Comment:
Why `21450`? It appears in the documentation, and then seems significant 🤔

Reply:
> At 21,450 Pages, the Longest Book in the World Is Impossible to Read. 

;-)